hw/ip/pwrmgr/dv/pwrmgr_sim_cfg.hjson

```diff
@@ -84,7 +88,41 @@
       name: pwrmgr_aborted_low_power
       uvm_test_seq: pwrmgr_aborted_low_power_vseq
     }
-
+    {
+      name: pwrmgr_sec_cm_lc_ctrl_intersig_mubi
+      uvm_test_seq: pwrmgr_repeat_wakeup_reset_vseq
+      run_opts: ["+test_timeout_ns=2000000", "+pwrmgr_mubi_mode=1"]
```

Comment:
you can use enum name instead of value, which is more readable.

Reply:
rewrote with enum.